frappe/docs/user/en/guides/integration/google_gsuite.md

```diff
@@ -0,0 +1,68 @@
+# Google GSuite
+
+You can create and attach Google GSuite Docs to your Documents using your predefined GSuite Templates.
+These Templates could use variables from Doctype that will be automatically filled.
+
+## Enable integration with Google Gsuite
+
+### Publish Google apps script
+
+1. Go to [https://script.google.com](https://script.google.com)
+1. Create a new Project. Click on **File > New > Project**
+1. Fill Code.gs the content of ***<bench dir>/apps/frappe/integrations/doctype/gsuite_settings/gsuite_settings.gs***
```

Comment:
I'm not asking to change any file, maybe I should move te .gs file to the documentation. because what I'm asking is to copy the content of .gs file to script.google.com. .gs file is the template in this case.
Of course we couldn't ask a user to change any file on frappe.